modules/luci-mod-dashboard/luasrc/view/dashboard/index.htm

```diff
@@ -0,0 +1,371 @@
+	width: 21%;
+	overflow: hidden;
+	padding-left:0;
+	word-break: break-all;
+}
+
+.Dashboard .router-status-wifi .wifi-info .settings-info{
+	padding:1em 0 1em 0;
+}
+
+.Dashboard .router-status-wifi .wifi-info .devices-info .tr .td:nth-child(3) {
+	width: 22%;
+	overflow: hidden;
+	position: relative;
+	top: -3px;
+}
+
+.Dashboard .router-status-wifi .wifi-info .devices-info .tr .td:nth-child(5) {
+	width: initial;
+}
+
+.Dashboard .router-status-wifi .wifi-info > hr:last-child {
+	margin-bottom:0;
+}
+
+.Dashboard .router-status-wifi .wifi-info .devices-info .device-info .progress {
+	padding: 0;
+	width: 100%;
+	margin: 0;
+}
+
+.Dashboard .wifi-info .devices-info .table-titles {
+	border-bottom:1px solid rgba(0,0,0,.1);
+}
+
+/**
+ * Responsive 
+ **/
+@media screen and (min-width: 200px) and (max-width: 640px) {
+
+	.Dashboard .cbi-section-1 > .section-content {
+		padding-top:10px;
+	}
+
+	.Dashboard .section-content {
+		display:block;
+	}
+
+	.Dashboard .section-content > div{
+		padding: 1em;
+	}
+
+	.Dashboard .section-content > div:first-child {
+		margin-bottom:10px;
+	}
+
+	.Dashboard .section-content > div:nth-child(2) {
+		margin:0;
+	}
+
+	.Dashboard .router-status-self .router-status-info .settings-info {
+		padding:0;
+	}
+
+	.Dashboard .section-content .internet-status-info .settings-info {
+		display:block;
+	}
+
+	.Dashboard .section-content .internet-status-info .settings-info > div:first-child {
+		margin-bottom: 10px;
+		border-bottom: 1px solid rgba(0,0,0,.1);
+	}
+
+	.Dashboard .section-content .router-status-lan .devices-info .table-titles {
+		display:block;
+	}
+
+	.Dashboard .router-status-wifi .wifi-info .settings-info > div{
+		flex:1;
+	}
+
+	.Dashboard .section-content .router-status-lan .devices-info .table-titles .th:last-child{
+		padding-left: 70px;
+	}
+
+	.Dashboard .section-content .router-status-lan .devices-info .td:first-child{
+		flex: 2 2 31%;
+	}
+
+	.Dashboard .section-content .router-status-lan .devices-info .td:nth-child(2){
+		flex: 1 1 24%;
+		padding: 0;
+	}
+
+	.Dashboard .section-content .router-status-lan .devices-info .td:last-child{
+		word-wrap: normal;
+	}
+
+	.Dashboard .router-status-wifi .wifi-info .settings-info > div p:nth-child(6) > span:last-child{
+		display: inline-block;
+		overflow: hidden;
+		height: 14px;
+		width: 52%;
+		word-break: break-word;
+		line-height: 15px;
+	}
+
+	.Dashboard .wifi-info .devices-info .table-titles {
+		padding: 0;
+		margin: 0;
+		display: flex;
+		border-radius: initial;
+	}
+
+	.Dashboard .wifi-info .devices-info .table-titles .th {
+		flex: 2 2 24%;
+	}
+
+	.Dashboard .wifi-info .devices-info .tr .td {
+		flex: 2 2 10%;
+	}
+
+	.Dashboard .wifi-info hr:nth-child(4) {
+		margin-bottom: 0;
+	}
+}
+
+]] -%>
+
+<%+header%>
+
+<div id="view">
+	<div class="spinning"><%:Loading view…%></div>
+</div>
+
+<script type="text/javascript">
+	L.require('ui').then(function(ui) {
+		ui.instantiateView('dashboard/index');
+	});
+</script>
+
+<%+footer%>
```

Comment:
Why is this code required? Would it be possible to remove all LUA dependencies?

Reply:
Because I need to use template `htm` to load custom css code, no other effective method has been found for the time being.

Do you have any suggestions ?

Reply:
Did you try something like that?

```javascript
document.querySelector('head').appendChild(E('link', {
    'rel': 'stylesheet',
    'type': 'text/css',
    'href': L.resource('style.css')
});
```

Reply:
Thanks for reminding! Very good method :) @jow-